### DISCUSSION TITLE
How go check if a third party module is compatible out of the box with breeze

### DISCUSSION THREAD

**Discussion by gfomengia** [breezefront/community#24] [5. General]
I have some third party module installed ( amsdty improved layered navigation). The different components are displayed on the frontend as require  y default. 
According to the breeze documentation, all third party js is removed. 

What will be your method to check?

**Comment by vovayatsyuk** [comment]
All modules that use js written in luma way (requirejs calls - require(), define()) requires integration. Otherwise, js-based features will not work in breeze.

In your particular case, I think ajax reload will not work when clicking on layered filters.
